src/tools/debugger/FrontendConnector.java

```diff
@@ -266,6 +371,41 @@ public void shutdown() {
     try {
       int delayMsec = 1000;
       receiver.stop(delayMsec);
-    } catch (InterruptedException e) { }
+    } catch (InterruptedException e) {
+
+    }
+  }
+
+/**
+ * Encapsulates the holder class and the method name of the breakpoint.
+ *
+ * @author carmentorres
+ *
+ */
+  public class BreakpointLocation{
+    private String holderClass;
```

Comment:
This is problematic. The class name is not a unique identifier. Would be good to have another, more unique, criterium.